frontend/src/services/wallet.js

```diff
@@ -36,65 +40,73 @@ export const checkForCRMToken = async (walletAddress) => {
     }
   } catch (error) {
     console.error("Error checking CRM token balance:", error);
-    throw error; // Ensures test will catch errors as thrown
-  }
-};
-
-export const getWallet = async () => {
-  if (globalWallet && globalWallet.isConnected) {
-    return globalWallet;
+    throw error;
   }
-
-  console.log('No wallet found. Attempting to connect...');
-  return await connectWallet(); // Fallback to connectWallet if not already connected
 };
 
-export const connectWallet = async () => {
-  try {
-    console.log('Attempting to connect to wallet...');
+  return checkForCRMToken;
+}
 
-    const { wallet } = await connect({
-      modalMode: "alwaysAsk",
-      modalTheme: "light",
-    });
 
-    if (!wallet) {
-      console.error('No wallet object found');
-      throw new Error('Failed to connect to wallet');
+export const useConnectWallet1 = () => {
```

Comment:
`useConnectWallet1` rename it pleae
```suggestion
export const useConnectWallet = () => {
```

Reply:
done